getting_started/step_by_step/your_first_game.rst

```diff
@@ -766,6 +766,11 @@ well as its position.
 Note that a new instance must be added to the scene using
 ``add_child()``.
 
+Now click on ``MobTimer`` in the scene window then head to inspector window,
```

Comment:
This seems like a strange place for this. The previous paragraph mentions `_on_MobTimer_timeout()`, which is created when you connect the signal. I understand you want to be more explicit about connecting the signal, but if you're going to add this, it should be earlier. Probably around line 729 where the three connections are made, but only the other two are explicitly described.

Reply:
@cbscribe  you have a point right there and i totally agree with you, but for me i think its more natural ( for me to understand and follow along with the tutorial without getting lost or confused ) to put it this way as after it there would be the step of actually writing the code for that function.

Reply:
OK, but then to keep the tutorial's flow, you'd have to do more rewriting than just inserting this paragraph.

You also have to change the text above in the two places:  1) line 729, where you're already told to create the connection (after your change you're now being told twice, which is arguably more confusing), and 2) line 760, where the connected method is mentioned as already existing. It's not "natural" to discuss these things out of order. 

As it is written, you connect all three signals, then fill each of them out one at a time. The mob signal just has more description inserted because more is going on. Maybe I could see moving the description until after the code, but MobTimer is no different from StartTimer or ScoreTimer, which are filled out the same way.

Reply:
i have made a new commit and it added a little hint to avoid the confusion on line 729 instead of inserting that paragraph on line 769, does it make it better now ?